build.sbt

```diff
@@ -86,6 +86,34 @@ lazy val s3Connector = project
     Test / fork := true
   )
 
+lazy val ftpConnector = project
+  .in(file("connectors/ftp-connector"))
+  .settings(stdSettings("zio-connect-ftp"))
+  .settings(
+    libraryDependencies ++= Seq(
+      FtpDependencies.zioFtp,
+      FtpDependencies.testContainersScala,
+      `zio`,
+      `zio-streams`,
+      `zio-prelude`,
```

Comment:
Can you move zio-prelude to FtpDependencies please. While it's an RC version I would like to avoid putting it in the core dependencies.